lib/bundler/rubygems_ext.rb

```diff
@@ -16,8 +16,15 @@ module Gem
   class Specification
     attr_accessor :remote, :location, :relative_loaded_from
 
-    remove_method :source if instance_methods(false).include?(:source)
-    attr_accessor :source
+    if instance_methods(false).map(&:to_s).include?("source")
```

Comment:
What's the context behind the addition of the type conversion here? (mapping to an array of strings now instead of symbols)